src/loaders/VolumeLoaderUtils.ts

```diff
@@ -13,6 +13,7 @@ const UNIT_SYMBOLS = {
   hour: "h",
   inch: "in",
   meter: "m",
+  micron: "μm",
```

Comment:
micrometer is handled in parts: by finding "meter" and turning it into "m," finding "micro" and turning it into "μ," and gluing them together. I just added this because I saw that "micron" was being applied as the default unit name in `TiffLoader` and thought we ought to be able to handle it, even though it's nonstandard according to OME-NGFF.